### DISCUSSION TITLE
Enable email Llanguage change based on frontend language

### DISCUSSION THREAD

**Discussion by mart-so** [supabase/supabase#27824] [Feature Requests]
I am currently developing a system that caters to users across multiple regions. To ensure a seamless user experience, I have implemented the frontend using i18n-react, which handles language localization effectively. However, I am facing a challenge regarding the synchronization of email language with the frontend language.

I have conducted preliminary research but have not found a built-in solution or documentation on how to dynamically set the language of emails based on the frontend language selection. Therefore, I am reaching out to inquire if this feature already exists or if it could be considered for future development.

Before proceeding with implementation, I would appreciate clarification on the following points:

Confirmation that there is no existing hidden feature that addresses this issue.
Guidance on the recommended approach for implementing this feature effectively.
I am willing to collaborate on developing this functionality once I have a clear understanding and feedback from your team.

Thank you for considering my request.

**Comment by GaryAustin1** [comment]
There is a hack: https://github.com/orgs/supabase/discussions/21227

Also you can use generateLink https://supabase.com/docs/reference/javascript/auth-admin-generatelink to build your own email on a server/edge function and call your own email provider thru SMTP or API.
